graphql/schema/types/version.graphql

```diff
@@ -69,6 +69,7 @@ type Version {
   taskStatusStats(options: BuildVariantOptions!): TaskStats
   upstreamProject: UpstreamProject
   versionTiming: VersionTiming
+  externalPatchLinks: [ExternalPatchLink!]!
```

Comment:
`externalPatchLinks` reads to me like it's a link to an external patch. Can we please change it to just externalLink or externalLinksForPatch? 
Similarly, can we please update the PR title and description to "external links for patches" or some other description that makes the distinction clear?

Reply:
+1

Reply:
I see that you made the change to the field. Can you please also change it in the PR title and description so that it's easy to understand if we ever need to refer back to it?